### ISSUE TITLE
Duplicate figure in Chapter 2 and Chapter 6

### ISSUE THREAD

**Issue by Robinlovelace** [geopython/geopython-workshop#104]
Just scanning through the content in preparation for the workshop tomorrow and really looking forward to it.

I noticed that the same image is used in 2 places, in Chapter 2:

![image](https://user-images.githubusercontent.com/1825120/185810617-8b4b8a02-3968-4501-bda1-2055d86117b2.png)

And Chapter 6:

![image](https://user-images.githubusercontent.com/1825120/185810640-26d0b22e-36d4-44f1-a165-80b670154917.png)

In Chapter 6 there could be a link back to Chapter 2 with text along the lines of "as shown in [Chapter 2](http://127.0.0.1:8888/notebooks/02-geometry.ipynb#Exploring-spatial-relationships-with-Shapely), ..."

Happy to have a go at this to remove duplicated content but also get that this could be deliberate. Many thanks for the materials and looking forward to it in any case!

**Comment by justb4** [comment]
Indeed, my suggestion is to refer to Chapter 2 in the text in Chapter 6 like @Robinlovelace indicates, "as shown in ...". Not removing it. Power of repetition in education...
